src/runtime/pkg/katautils/config.go

```diff
@@ -1104,7 +1105,14 @@ func LoadConfiguration(configPath string, ignoreLogging bool) (resolvedConfigPat
 	}
 
 	config.Debug = tomlConf.Runtime.Debug
-	if !tomlConf.Runtime.Debug {
+	config.LogLevel, err = logrus.ParseLevel(tomlConf.Runtime.LogLevel)
+	if err != nil {
+		return "", oci.RuntimeConfig{}, err
+	}
+
+	if tomlConf.Runtime.Debug {
+		kataUtilsLogger.Logger.Level = config.LogLevel
```

Comment:
This is neat as it doesn't break existing users but it's slightly confusing because you need to enable debug to change the level to something other than debug.

I wonder if instead we should just deprecate `enable_debug`. Something like:

```go
if tomlConf.Runtime.LogLevel != "" {
    kataUtilsLogger.Logger.Level = config.LogLevel
} else if tomlConf.Runtime.Debug {
     kataUtilsLogger.Logger.Level = Level.DebugLevel
}
```

If we do something like that shown above, we should also display a warning message similar to:

```
'enable_debug=' is deprecated - use 'loglevel=' instead"
```

... but note that actually outputting that message needs care since:

- This code is setting up the log levels and we always want that message to be displayed.
- Displaying the message "too early" will upset the container manager, which expects certain early output from the runtime and will fail if it sees something it doesn't expect.

We'd also need to think carefully about actually _removing_ `enable_debug=` since it would need changes to atleast:

- Docs.
- A bunch of scripts in the tests repo.

wdyt @amshinde, @bergwolf, @fidencio, @egernst?

Reply:
@jodh-intel  +1 from me on removing enable_debug altogether although we may need to make quite a few changes in our logs and testing scripts. Having the ability to set the log level as added by this PR is of value to the user but having the enable_debug option is quite confusing and may conflict with the log level set by the option introduced in this PR.

Reply:
@jodh-intel @fidencio I agree with replacing `enable_debug` config with `loglevel` config. If all committers reach an agreement with this change, I will modify the patch to support this change.

Reply:
@jodh-intel, worries me quite a bit we remove an option and add a totally different one in between **minor** releases.

This may break compatibility for users who have their custom config / custom scripts to change those knobs in their nodes.

I sincerely think something as huge as this should be targeting a **major** release instead, being material for 3.0 rather than 2.x.